gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/Misc.java

```diff
@@ -761,6 +762,11 @@ public static int getUnifiedHashCodeFromDVD(DataValueDescriptor[] dvds,
     }
   }
 
+  public static boolean isSecurityEnabled() {
+    return "LDAP".equalsIgnoreCase(getMemStore().getBootProperty(Attribute.AUTH_PROVIDER)) ||
+        "LDAP".equalsIgnoreCase(getMemStore().getBootProperty(Attribute.SERVER_AUTH_PROVIDER));
+  }
```

Comment:
@ashetkar Why not simply use the method added below: `AuthenticationServiceBase authService = AuthenticationServiceBase.getAuthenticationServiceBase(); return authService != null && !(authService instanceof NoneAuthenticationServiceImpl)`